generate-examples/src/index.ts

```diff
@@ -20,6 +19,10 @@ interface BundleInfo {
   types: string[];
 }
 
+// revocation data
+const issuerSupportingRevocation = new Set([0]); // only one issuer (kid) supports it
+const revocationHmacSecretKey = crypto.randomBytes(32); // 256-bit HMAC key for calculating revocation IDs
```

Comment:
Should we grab this from config the way we grab private keys (e.g., https://github.com/smart-on-fhir/health-cards/blob/main/generate-examples/src/config/issuer.jwks.private.json)? Otherwise won't the `rid` in our example SHCs change each time the generation script runs?

Reply:
Yes, `rid` values would change each time, but they would be updated everywhere, and wouldn't affect card processing. We can certainly pre-generate the secret to keep these values static.

Reply:
Yeah, I think using a static HMAC secret would give us a way to publish examples of revoked cards.

Reply:
My point was that when you publish the revoked examples and CRLs, they would have consistent rid values even if different from run to run, because the script re-generate all of these artefacts at the same time. Moot point, I updated the code to use a static secret.